paradox-nakadi-consumer-sqs-error-handler/src/main/java/de.zalando.paradox.nakadi.consumer.sqserrorhandler/SQSErrorHandler.java

```diff
@@ -25,10 +26,13 @@
 
     private final ObjectMapper objectMapper;
 
+    private final String queueUrl;
+
     public SQSErrorHandler(final SQSConfig sqsConfig, final AmazonSQS amazonSQS, final ObjectMapper objectMapper) {
         this.amazonSQS = amazonSQS;
         this.queueName = sqsConfig.getQueueName();
         this.objectMapper = objectMapper;
+        this.queueUrl = amazonSQS.getQueueUrl(queueName).getQueueUrl();
```

Comment:
I am not sure whether in Amazon sdk, it has a state or not. If it has a state in Amazon side, it may cause some problems otherwise, it is good to cache it.